README.md

```diff
@@ -82,7 +82,9 @@ module.exports = {
         matchFields: ['slug', 'modified'], // Array<String> default: ['modified']
         concurrentQueries: false, // default: true
         skipIndexing: true, // default: false, useful for e.g. preview deploys or local development
-        continueOnFailure: false // default: false, don't fail the build if algolia indexing fails
+        continueOnFailure: false, // default: false, don't fail the build if algolia indexing fails
+        useEnvironment: 'blog', // default: false, used in conjunction with enablePartialUpdates let you persist entries from different environment
+        environmentKey: 'product', // default: 'environment', key used when useEnvironment is set.
```

Comment:
this seems confusing, what if we have only the option `environmentKey`, and use it whenever it's set. I think also the option on line 86 possibly has the wrong example value? (is it a boolean or a string?)